components/qqr.js

```diff
@@ -0,0 +1,1198 @@
+// [qr.js](http://neocotic.com/qr.js)  
+// (c) 2014 Alasdair Mercer  
+// Licensed under the GPL Version 3 license.  
```

Comment:
Talked to Othe about using this library: https://github.com/fukuchi/libqrencode
Although we have to add it as a dependency and not include it in the repo.